esphome/components/abbaurora/abbaurora.cpp

```diff
@@ -0,0 +1,1034 @@
+    case 18:
+      return std::string("Internal Error E027");
+    case 19:
+      return std::string("Internal Error E028");
+    case 20:
+      return std::string("Internal Error E029");
+    case 21:
+      return std::string("Internal Error E030");
+    case 22:
+      return std::string("Sending Wind Table");
+    case 23:
+      return std::string("Failed Sending table");
+    case 24:
+      return std::string("UTH Fault");
+    case 25:
+      return std::string("Remote OFF");
+    case 26:
+      return std::string("Interlock Fail");
+    case 27:
+      return std::string("Executing Autotest");
+    case 30:
+      return std::string("Waiting Sun");
+    case 31:
+      return std::string("Temperature Fault");
+    case 32:
+      return std::string("Fan Staucked");
+    case 33:
+      return std::string("Int.Com.Fault");
+    case 34:
+      return std::string("SLV Insertion");
+    case 35:
+      return std::string("DC Switch Open");
+    case 36:
+      return std::string("TRAS Switch Open");
+    case 37:
+      return std::string("MST Exclusion");
+    case 38:
+      return std::string("Auto Exclusion");
+    case 98:
+      return std::string("Erasing Internal EEprom");
+    case 99:
+      return std::string("Erasing External EEprom");
+    case 100:
+      return std::string("Counting EEprom");
+    case 101:
+      return std::string("Freeze");
+    default:
+      ESP_LOGD(TAG, "GlobalState Text unknown [%d]", id);
+      return std::string("Unknown");
+  }
+}
+
+std::string ABBAuroraComponent::dcdc_state_text(uint8_t id) {
+  switch (id) {
+    case 0:
+      return std::string("DcDc OFF");
+    case 1:
+      return std::string("Ramp Start");
+    case 2:
+      return std::string("MPPT");
+    case 3:
+      return std::string("Not Used");
+    case 4:
+      return std::string("Input OC");
+    case 5:
+      return std::string("Input UV");
+    case 6:
+      return std::string("Input OV");
+    case 7:
+      return std::string("Input Low");
+    case 8:
+      return std::string("No Parameters");
+    case 9:
+      return std::string("Bulk OV");
+    case 10:
+      return std::string("Communication Error");
+    case 11:
+      return std::string("Ramp Fail");
+    case 12:
+      return std::string("Internal Error");
+    case 13:
+      return std::string("Input mode Error");
+    case 14:
+      return std::string("Ground Fault");
+    case 15:
+      return std::string("Inverter Fail");
+    case 16:
+      return std::string("DcDc IGBT Sat");
+    case 17:
+      return std::string("DcDc ILEAK Fail");
+    case 18:
+      return std::string("DcDc Grid Fail");
+    case 19:
+      return std::string("DcDc Comm.Error");
+    default:
+      ESP_LOGD(TAG, "DcDcState Text unknown [%d]", id);
+      return std::string("Unknown");
+  }
+}
+
+std::string ABBAuroraComponent::inverter_state_text(uint8_t id) {
+  switch (id) {
+    case 0:
+      return std::string("Stand By");
```

Comment:
```suggestion
      return "Stand By";
```

The `std::string` mapping isn't needed, as it's automatically converted by the implicit constructor